doc/source/notebooks/basics/classification.ipynb

```diff
@@ -537,20 +540,14 @@
     "There are dedicated notebooks giving more details on how to manipulate [models](../understanding/models.ipynb) and [kernels](../advanced/kernels.ipynb).\n",
```

Comment:
This links to understanding/models.ipynb which doesn't exist ... thoughts !? what kind of content were you expecting to go in there ?

Reply:
This is linked in another place in this notebook, too